src/easylink/pipeline_schema.py

```diff
@@ -21,23 +22,20 @@ def __repr__(self) -> str:
     def set_step_config(self, parent_config: LayeredConfigTree) -> None:
         self._config = parent_config
 
-    def get_pipeline_graph(self, pipeline_config: LayeredConfigTree) -> nx.MultiDiGraph:
+    def get_pipeline_graph(self, pipeline_config: LayeredConfigTree) -> ImplementationGraph:
         """Resolve the PipelineSchema into a PipelineGraph."""
-        graph = nx.MultiDiGraph()
-        graph.add_node(self.name, step=self)
-        self.update_implementation_graph(graph)
-        return graph
+        return self.get_implementation_graph()
```

Comment:
I think `get_pipeline_graph` being the same as `get_implementation_graph` adds more confusion to what the difference is between the two classes.

Reply:
per your and Rajan's comments I just deleted this method entirely